### PR TITLE
Guide on basic DB operations (close #2092)

### LINKED ISSUE #2092: docs: guide on handling basic database operations with hasura
1. How to open psql on heroku, digitalocean one-click and other providers
2. How to export/import data
3. How to export/import schema
4. How to do data backups

### PR DESCRIPTION
**Pull request by marionschleifer**
### Description
For Hacktoberfest, different people want to contribute to this issue: https://github.com/hasura/graphql-engine/issues/2092. 

Contributors can branch off this branch and add sections that they want to add. I will review and we can merge in this branch. Finally we can merge to master.

### Affected components 
- Docs

### CODE REVIEW

File: docs/graphql/manual/postgres/psql/heroku.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Using psql usning Heroku
```
Comment (utkarsh2102) [inline docs/graphql/manual/postgres/psql/heroku.rst:1]:
```suggestion
Using psql using Heroku
```

**Review by utkarsh2102 — COMMENTED** [review]
Hi @marionschleifer,

A small typo fix below, you can just commit from here, directly ⬇️

**Review by codeliezel — APPROVED** [review]
Great!

**Comment by marionschleifer** [comment]
Closing this as it's completely outdated with structure & content.

**Pull request closed without merging**